File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.LookAndFeel;
using DevExpress.XtraEditors.Controls;
using DevExpress.Skins;

namespace WindowsApplication1
{
    public partial class Form1 : Form
    {
        private DataTable CreateTable(int RowCount)
        {
            DataTable tbl = new DataTable();
            tbl.Columns.Add("Name", typeof(string));
            tbl.Columns.Add("ID", typeof(int));
            tbl.Columns.Add("Number", typeof(int));
            tbl.Columns.Add("Date", typeof(DateTime));
            for (int i = 0; i < RowCount; i++)
                tbl.Rows.Add(new object[] { String.Format("Name{0}", i), i, 3 - i, DateTime.Now.AddDays(i) });
            return tbl;
        }

        public Form1()
        {
            InitializeComponent();
            gridControl1.DataSource = CreateTable(20);
            foreach (var item in Enum.GetValues(typeof(LookAndFeelStyle)))
            {
                rgStyles.Properties.Items.Add(new RadioGroupItem(item, item.ToString()));
            }
            foreach (SkinContainer item in SkinManager.Default.Skins)
            {
                rgSkins.Properties.Items.Add(new RadioGroupItem(item.SkinName, item.SkinName));
            }
        }

        private void rgStyles_SelectedIndexChanged(object sender, EventArgs e)
        {
            UserLookAndFeel.Default.Style = (LookAndFeelStyle)rgStyles.EditValue;
        }

        private void rgSkins_SelectedIndexChanged(object sender, EventArgs e)
        {
            UserLookAndFeel.Default.SkinName = rgSkins.EditValue.ToString();
            rgStyles.EditValue = LookAndFeelStyle.Skin;
            
        }

        
    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.LookAndFeel
Imports DevExpress.XtraEditors.Controls
Imports DevExpress.Skins

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits Form
		Private Function CreateTable(ByVal RowCount As Integer) As DataTable
			Dim tbl As New DataTable()
			tbl.Columns.Add("Name", GetType(String))
			tbl.Columns.Add("ID", GetType(Integer))
			tbl.Columns.Add("Number", GetType(Integer))
			tbl.Columns.Add("Date", GetType(DateTime))
			For i As Integer = 0 To RowCount - 1
				tbl.Rows.Add(New Object() { String.Format("Name{0}", i), i, 3 - i, DateTime.Now.AddDays(i) })
			Next i
			Return tbl
		End Function

		Public Sub New()
			InitializeComponent()
			gridControl1.DataSource = CreateTable(20)
			For Each  style As Object In System.Enum.GetValues(GetType(LookAndFeelStyle))
				rgStyles.Properties.Items.Add(New RadioGroupItem(style, style.ToString()))
			Next style
			For Each item As SkinContainer In SkinManager.Default.Skins
				rgSkins.Properties.Items.Add(New RadioGroupItem(item.SkinName, item.SkinName))
			Next item
		End Sub

		Private Sub rgStyles_SelectedIndexChanged(ByVal sender As Object, ByVal e As EventArgs) Handles rgStyles.SelectedIndexChanged
			UserLookAndFeel.Default.Style = CType(rgStyles.EditValue, LookAndFeelStyle)
		End Sub

		Private Sub rgSkins_SelectedIndexChanged(ByVal sender As Object, ByVal e As EventArgs) Handles rgSkins.EditValueChanged
			UserLookAndFeel.Default.SkinName = rgSkins.EditValue.ToString()
			rgStyles.EditValue = LookAndFeelStyle.Skin

		End Sub


	End Class
End Namespace
